### ISSUE TITLE
Twitter - Hash tag search not returning expected results

### ISSUE THREAD

**Issue by NickStees** [Fastspot/bigtree-social-feed#2]
When creating a hashtag query for "#getclever" I don't get any returned results but searching on Twitter there are clearly hashtags there https://twitter.com/hashtag/GetClever?src=hash but when I change the query to "#coffee" I seem to get results. Does this hashtag query only search the connected account? It seems not to to be the case since "#coffee" returns other peoples tweets. I just can't figure out why I don't get results for the first query.

**Comment by timbuckingham** [comment]
My best guess here is that Twitter's API only returns results that are fairly recent and the most recent tweet with that hashtag is over a month old. Using the Twitter API Console (https://dev.twitter.com/rest/tools/console) gives no results for the search query either so it's out of my control unfortunately.

I did a test by tweeting from one of my dev accounts with the hashtag and it properly returned that tweet (around 60 seconds or so after I posted it the API started returning it). I'm not sure at which age the API will ignore a tweet but there must be a cutoff.
